### PR TITLE
Skios 176 subscription page not updated after a new feed is subscribed

### PR DESCRIPTION
**Pull request by LukasLJL**
# Skiosa - PR
- [x] added test(s), or explain why one is not needed/possible...
- [x] added comments to all the functions
- [ ] added documentation to latex project
- [x] code can't be split up into smaller logical units
- [x] code is formated and linted

PR Credits @grossamos

### CODE REVIEW

**Comment by simonmorgenstern** [comment]
why do we need the module? which problem is solved by using a module instead of a component itself?

**Comment by grossamos** [comment]
> why do we need the module? which problem is solved by using a module instead of a component itself? 
> 

Module binds Page and nessicary components, I find this to be an improvement.

**Comment by simonmorgenstern** [comment]
> > why do we need the module? which problem is solved by using a module instead of a component itself?
> 
> Module binds Page and nessicary components, I find this to be an improvement.

fair enough, at least it contains 2 components :P

**Comment by grossamos** [comment]
Perfect! What's the status on this review btw?

**Comment by simonmorgenstern** [comment]
> Perfect! What's the status on this review btw?

it is in review? :)

**Comment by grossamos** [comment]
it would be great to have all stories approved for the review today (this one has been do that have been done for a couple of days)

File: src/app/pages/subscription-page/subscription-feed/subscription-feed.component.html [hunk 1]
```diff
@@ -2,16 +2,16 @@
 	<div
 		class="subscription-feed-icon-background"
 		[ngStyle]="{ 'background-color': color }"
+		[routerLink]="'/feed/' + feed.id"
```
[the remark this answers is not in the document]
Comment (simonmorgenstern) [reply src/app/pages/subscription-page/subscription-feed/subscription-feed.component.html:5]: will be fixed in SKIOS-169

**Review by simonmorgenstern — APPROVED** [review]
approved, refactor requests will be moved to sepearate ticket due to missign time

**Pull request merged**